pkgs/development/python-modules/withings-api/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, poetry-core
+, arrow
+, requests-oauthlib
+, typing-extensions
+, pydantic
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "withings-api";
+  version = "2.4.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "withings_api";
+    inherit version;
+    sha256 = "sha256-vQ6MKeD4g4QTkXx638FW53mTkx78af7NQXF00kxgM10=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'requests-oauth = ">=0.4.1"' '''
```

Comment:
Can we also report this issue upstream?

Reply:
https://github.com/vangorra/python_withings_api/issues/90